homeassistant/components/camera/__init__.py

```diff
@@ -681,10 +680,9 @@ def async_update_token(self) -> None:
     async def async_internal_added_to_hass(self) -> None:
         """Run when entity about to be added to hass."""
         await super().async_internal_added_to_hass()
-        # Note: State is always updated by entity on return
         await self.async_refresh_providers()
```

Comment:
If we want to avoid the double state update we could replace this line with line 694.

Reply:
OK, i'll opt to leave it for now and optimize for simpler code structure given this will be rarely used.